docs/setup-docs.md

```diff
@@ -1,9 +1,9 @@
-# Setting up a Federation + Lightning Gateway on Mutinynet
+# Setting up a Federation + Lightning Gateway
 
-This is a guide for setting up a 3/4 federation + lightning gateway on mutinynet, a test network for bitcoin with 30 second block times. You can run this same setup on a different test network or on mainnet with real bitcoin, but that is not encouraged while fedimint remains under active development.
+This is a guide for setting up a 3/4 federation + lightning gateway, a test network for bitcoin with 30 second block times. You can run this same setup on a different test network or on mainnet with real bitcoin,
```

Comment:
If you remove `mutinynet` then the thing about 30s block times is also not relevant anymore.